k8s/cmd/commands/create/create_backup.go

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 package create
 
 import (
+	"log"
```

Comment:
Use your `util.ErrLogger`. Mixing different logger in the same program is generally a bad practice.

Reply:
My bad. Thanks for pointing that out.